tinlake-ui/containers/UserRewards/index.tsx

```diff
@@ -325,16 +325,16 @@ const Explainer = () => (
       <HelpIcon src="/static/help-circle.svg" />
       <h3 style={{ margin: 0 }}>How it works</h3>
     </Box>
-    Radial (RAD) Rewards are earned on Ethereum based on your Tinlake investments but claimed on Centrifuge Chain. To
-    claim your rewards you need to link your Tinlake investment account to a Centrifuge Chain account receiving and
-    holding your RAD.
+    Centrifuge (CFG) Rewards are earned on Ethereum based on your Tinlake investments but claimed on Centrifuge Chain.
```

Comment:
This reads a bit weird, can we just say:

```suggestion
    CFG rewards are earned on Ethereum based on your Tinlake investments but claimed on Centrifuge Chain.
```